posts/centralized-rules-execution-server.md

```diff
@@ -0,0 +1,100 @@
+---
+authors:
+- Mallik Sambaraju
+- Amine Ouali Alami
+tags:
+- Rules
+- Kie
+- Drools
+- JHipster
+- AWS ECS
+- Spring Boot
+date: 2010-01-23T10:00:00.000Z
+title: Centralized Rules Engine on AWS ECS
+---
+
+## Introduction
+As software engineers we architect and develop many real world applications solving many business problems, and in those applications we add business logic to solve various issues. Often this business logic is difficult to implement in application code, business rules keep on changing and hard to maintain.
```

Comment:
Often this business logic is difficult to implement in application code and business rules keep changing which is hard to maintain.